rdfhash/utils/__init__.py

```diff
@@ -2,6 +2,37 @@
 
 from .hash import hash_types
 
+triple_mime_types = {
+    "text/turtle",
+    "application/n-triples",
+}
+
+rdf_ext_to_mime_types = {
+    "ttl": "text/turtle",
+    "nt": "application/n-triples",
+    "trig": "application/trig",
+    "nq": "application/n-quads",
+    "rdf": "application/rdf+xml",
+}
+rdf_mime_types = {
+    "text/turtle",
+    "application/n-triples",
+    "application/trig",
+    "application/n-quads",
+    "application/rdf+xml",
+}
+
+
+def get_rdf_mime_type(format: str):
+    format = format.lower()
+    mime_type = rdf_ext_to_mime_types.get(format, format)
+    if mime_type not in rdf_mime_types:
+        raise ValueError(
+            "Invalid RDF format: {format}. Must be one of {rdf_mime_types}.".format(
+                format=format, rdf_mime_types=rdf_mime_types
+            )
+        )
```

Comment:
This doesn't appear to return `mime_type` when successful. Code should be something like
```python
     try:
         return rdf_ext_to_mime_types[format.lower()]
     except KeyError:
        raise ValueError(
            f"Unrecognized file extension: {format}. Must be one of {rdf_mime_types}."
        )
```